src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,43 @@
+import { Link } from "react-router-dom";
+import logo from "../../assets/logo.png"
```

Comment:
use absolute path for this to avoid ../../

Reply:
will read about this